components/eamxx/tests/coupled/dynamics_physics/homme_shoc_cld_p3_rrtmgp/CMakeLists.txt

```diff
@@ -61,6 +61,7 @@ configure_file(${SCREAM_SRC_DIR}/dynamics/homme/tests/theta.nl
 
 # Ensure test input files are present in the data dir
 GetInputFile(init/${EAMxx_tests_IC_FILE_72lev})
+GetInputFile(../cam/topo/USGS-gtopo30_ne4np4_16x.c20160612.nc)
```

Comment:
Can we use `${TOPO_DATA_DIR}` instead of `../cam/topo`, here and also in the other CMakeLists files?

Reply:
Ah, I see now. I think the change of the base dir for the search path is a good idea.